docs/module_guides/models/multi_modal.md

```diff
@@ -32,9 +32,47 @@ response = openai_mm_llm.complete(
 )
 ```
 
+**Legend**
+
+- ✅ = should work fine
+- 🛑 = not available at the moment. Support on the way
+
+### End to End Multi-Modal Work Flow
+
+The tables below attempt to show the **initial** steps with various LlamaIndex features for building your own Multi-Modal RAGs. You can combine different modules/steps together for composing your own Multi-Modal RAG orchestration.
+
+| Query Type | Data Sources<br>for MultiModal<br>Vector Store/Index | MultiModal<br>Embedding                | Retriever                                        | Query<br>Engine        | Output<br>Data<br>Type                   |
+| ---------- | ---------------------------------------------------- | -------------------------------------- | ------------------------------------------------ | ---------------------- | ---------------------------------------- |
+| Text ✅    | Text ✅                                              | Text ✅                                | Top-k retrieval ✅<br>Simple Fusion retrieval ✅ | Simple Query Engine ✅ | Retrieved Text ✅<br>Generated Text ✅   |
+| Image ✅   | Image ✅                                             | Image ✅<br>Image to Text Embedding ✅ | Top-k retrieval ✅<br>Simple Fusion retrieval ✅ | Simple Query Engine ✅ | Retrieved Image ✅<br>Generated Image 🛑 |
+| Audio 🛑   | Audio 🛑                                             | Audio 🛑                               | 🛑                                               | 🛑                     | Audio 🛑                                 |
+| Video 🛑   | Video 🛑                                             | Video 🛑                               | 🛑                                               | 🛑                     | Video 🛑                                 |
+
+### Multi-Modal LLM Models
+
+These notebooks serve as examples how to leverage and integrate Multi-Modal LLM model, Multi-Modal embeddings, Multi-Modal vector stores, Retriever, Query engine for composing Multi-Modal RAG orchestration.
+
+| Multi-Modal<br>Vision Models                                                                                                            | Single<br>Image<br>Reasoning | Multiple<br>Images<br>Reasoning | Image<br>Embeddings | Simple<br>Query<br>Engine |
+| --------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- | ------------------------------- | ------------------- | ------------------------- |
+| [GPT4V](https://github.com/run-llama/llama_index/blob/main/docs/examples/multi_modal/gpt4v_multi_modal_retrieval.ipynb)<br>(OpenAI API) | ✅                           | ✅                              | 🛑                  | ✅                        |
```

Comment:
nit: could we use relative links instead? e.g. [GPT4V](/examples/multi_modal/gpt4v_multi_modal_retrieval.ipynb), will link to the compiled doc